cmd/oras/internal/option/pusher.go

```diff
@@ -73,3 +93,11 @@ func decodeJSON(filename string, v interface{}) error {
 	defer file.Close()
 	return json.NewDecoder(file).Decode(v)
 }
+
+// validateEmptyManifestAnnotations check whether existing ManifestAnnotations contain no value.
+func validateEmptyManifestAnnotations(annotations map[string]map[string]string) error {
```

Comment:
We don't need an extra function here. `len(annotations[annotationManifest]) == 0` is enough. No need to check config annotation.